docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -63,6 +63,18 @@ The `terraform` block supports the following arguments:
       registry]({{site.baseurl}}/docs/getting-started/quick-start#a-note-about-using-modules-from-the-registry) for more
       information about using modules from the Terraform Registry with Terragrunt.
 
+- `include_in_copy` (attribute): A list of glob patterns (e.g., `*.txt`) that should always be copied into the Terraform
+  working directory. When you use the `source` param in your Terragrunt config and run `terragrunt <command>`,
+  Terragrunt will download the code specified at source into a scratch folder (`.terragrunt-cache`, by default), copy
+  the code in your current working directory into the same scratch folder, and then run `terraform <command>` in that
+  scratch folder. By default, Terragrunt excludes hidden files and folders during the copy step. This feature allows you
+  to specify glob patterns of files that should always be copied from the Terragrunt working directory. Additional notes:
+    - The path should be specified relative to the source directory.
+    - This list is also use when using a local file source (e.g., `source = "../modules/vpc"`). For example, if your
```

Comment:
```suggestion
    - This list is also used when using a local file source (e.g., `source = "../modules/vpc"`). For example, if your
```

Reply:
Done in https://github.com/gruntwork-io/terragrunt/pull/1941